src/current/_includes/releases/v22.2/v22.2.18.md

```diff
@@ -0,0 +1,22 @@
+## v22.2.18
+
+Release Date: February 8, 2024
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v22-2-18-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug that could cause [`DELETE`](https://www.cockroachlabs.com/docs/v22.2/delete.html) queries sent by the [row-level TTL job](https://www.cockroachlabs.com/docs/v22.2/row-level-ttl.html) to use a [secondary index](https://www.cockroachlabs.com/docs/v22.2/schema-design-indexes.html) rather than the [primary index](https://www.cockroachlabs.com/docs/v22.2/indexes.html) to find the rows to delete. This could lead to some `DELETE` operations taking longer than expected. This bug was present since v22.2.0. [#118420][#118420]
+- Fixed a bug where a [changefeed](https://www.cockroachlabs.com/docs/v22.2/change-data-capture-overview.html) could omit events in rare cases, logging the error `cdc ux violation: detected timestamp ... that is less or equal to the local frontier`. This could happen in the following scenario:
+    1. A [rangefeed](https://www.cockroachlabs.com/docs/v22.2/create-and-configure-changefeeds.html#enable-rangefeeds) runs on a follower [replica](https://www.cockroachlabs.com/docs/v22.2/architecture/glossary#cockroachdb-architecture-terms) that lags significantly behind the [leaseholder](https://www.cockroachlabs.com/docs/v22.2/architecture/glossary#cockroachdb-architecture-terms).
+    1. A transaction commits and removes its transaction record before its [intent](https://www.cockroachlabs.com/docs/v22.2/architecture/transaction-layer#writing) resolution is applied on the follower.
+    1. The follower's [closed timestamp](https://www.cockroachlabs.com/docs/v22.2/architecture/transaction-layer#closed-timestamps) has advanced past the transaction commit timestamp.
+    1. The rangefeed attempts to push the transaction to a new timestamp (at least 10 seconds after the transaction began).
+    1. This may cause the rangefeed to prematurely emit a checkpoint before emitting writes at lower timestamps, which in turn may cause the [changefeed](https://www.cockroachlabs.com/docs/v22.2/how-does-an-enterprise-changefeed-work) to drop these events entirely, never emitting them. [#118633][#118633]
```

Comment:
This link does not exist in v22.2: https://www.cockroachlabs.com/docs/v22.2/how-does-an-enterprise-changefeed-work

Reply:
Wow, ok didn't break the build 😬